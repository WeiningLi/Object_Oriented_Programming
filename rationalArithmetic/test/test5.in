te a 5/2
print a
create b 15/2
print b
+ a b
- b a
* a b
/ b a
+= a b
-= a b
u- a

