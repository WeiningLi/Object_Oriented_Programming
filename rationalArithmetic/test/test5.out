0
15/2
15/2
15/2
0
1/0
15/2
0
0
